packages/manager/modules/manager-pci-common/package.json

```diff
@@ -21,7 +21,8 @@
     "element-internals-polyfill": "^1.3.11",
     "file-saver": "^2.0.5",
     "i18next": "^23.8.2",
-    "i18next-http-backend": "^2.6.1"
+    "i18next-http-backend": "^2.6.1",
```

Comment:
as QA has been done on this branch we won't be able to separate the changes for now, i informed the team to add you and other team members in case of future changes in pci-common to the list of reviewers in PRs to make it easier to track new components & hooks